pyleecan/Functions/MeshSolution/get_indices_limited.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
I am not sure why you need this method. Is it possible to directly use get_group method ?

Reply:
It's because it only return indices of cells thats field values are within a given range.

Reply:
OK. It would be interesting to integrate this feature in get_group latter. ;-)